config/mvc-signrequest.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
```

Comment:
dit gaan we iets anders inrichten de nieuwe naam (naar onze service hernoemd) wordt:
mvc-document-sign.php

hier ziet de return er zo uit:
```php
return [
    'default' => 'signrequest',
    'providers' => [
        'signrequest' => [
                'token' => '',
                ...
        ]
    ]
]
```